### DISCUSSION TITLE
web pag 可以导出mp4码

### DISCUSSION THREAD

**Discussion by a6965921** [Tencent/libpag#82] [Q&A] [answered]
支不支持nodejs 运行js 然后导出图片

**Comment by zenoslin** [comment, marked as answer]
不支持的，可以通过录制webgl的方式导出mp4

**Reply by a6965921** [reply]
> 不支持的，可以通过录制webgl的方式导出mp4

支不支持nodejs 运行js 然后导出图片

**Reply by zenoslin** [reply]
Web 版是不支持的，因为用了很多 Browser 的能力。服务端渲染建议使用 Linux 接入方案，如果用 Node.js 兼容也只是胶水层调用 Native 的能力。

**Reply by a6965921** [reply]
> Web 版是不支持的，因为用了很多 Browser 的能力。服务端渲染建议使用 Linux 接入方案，如果用 Node.js 兼容也只是胶水层调用 Native 的能力。

后期可以支持下服务端渲染吗

**Reply by domchen** [reply]
服务端渲染是有的，不是通过Web平台实现。而是用C++的linux SDK。

**Comment by LostTong** [comment]
> 服务端渲染是有的，不是通过Web平台实现。而是用C++的linux SDK。

请问，linux cpp从pag转换为mp4使用哪个sdk？用ffmpeg？有没有 demo 或 代码链接 参考下？感谢~
